gulpfile.js

```diff
@@ -50,7 +39,7 @@ function build() {
     .pipe(gulp.dest('public'))
 }
 
-function watch() {
+function watch () {
```

Comment:
Hmm... it's detecting it as an error now, but it's not autofixing. Any idea why?